common/app/quiz/form/package.scala

```diff
@@ -27,6 +27,21 @@ package object form {
     }
   }
 
+  // Find the correct answer for a given question.
+  private[quiz] def getCorrectAnswer(question: Question): Option[Answer] = {
+    question.answers.find(_.weight == 1)
+  }
+
+  // Returns true of inputAnswer is the correct answer of inputQuestion.
+  def isCorrectAnswer(inputQuestion: Question, inputAnswer: Answer): Option[Boolean] = {
+    getCorrectAnswer(inputQuestion).map(_.id == inputAnswer.id)
+  }
+
+  // Returns true if each answer in this question has images.
+  def hasImages(question: Question): Boolean = {
+    !question.answers.exists(answer => answer.imageMedia.isEmpty)
```

Comment:
because "there exists something nonEmpty" isn't equal to "there does not exist anything empty". The first being an existential quantifier, but I really need a universal quantifier to ensure everything in this question has an image.